tidb-cloud/tidb-cloud-events.md

```diff
@@ -0,0 +1,55 @@
+---
+title: TiDB Cloud Events
+summary: Learn how to review the events for TiDB Cloud cluster using the Event page. 
+---
+
+# TiDB Cloud Events
+
+TiDB Cloud logs the historical events at the cluster level. An *event* indicates a change in the TiDB Cloud environment. You can review the logged events on the **Events** page, including the event type, status, message, trigger time, and trigger user.
+
+This document describes how to review the events for TiDB Cloud clusters using the **Events** page. 
+
+> **Note:**
+>
+> The Event page currently is only available for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.
```

Comment:
Can you add this limitation to this doc: [Serverless Tier Limitations](https://docs.pingcap.com/tidbcloud/serverless-tier-limitations)